src/libvfcfuncinstrument/libVFCFuncInstrument.cpp

```diff
@@ -58,24 +58,10 @@ static Function *func_enter;
 static Function *func_exit;
 
 // Enumeration of managed types
-enum Ftypes { FLOAT, DOUBLE };
+enum Ftypes { FLOAT, DOUBLE, FLOAT_PTR, DOUBLE_PTR };
 
 // Array of values
-Value *Types2val[] = {NULL, NULL};
-
-// Demangling function
-std::string demangle(std::string src) {
```

Comment:
@yohanchatelain are you ok with removing the demangle function ? I know we discussed this online, but I do not remember what we decided. I would like your feedback before we remove this functionality.

Reply:
I discussed with @yohanchatelain which is ok with this. Resolving